js-extensions/spec/ExtensionStore-spec.js

```diff
@@ -92,7 +92,7 @@ describe("ExtensionStore.js", function () {
             var extensionStore = makeExtensionStore(plugins, false);
 
             try {
-                extenstionStore.getExtensions('ext', function(ext) { });
+                extensionStore.getExtensions('ext', function(ext) { });
```

Comment:
funny, exception thrown... because of the misspelling. *sigh*

Reply:
It's a pretty common problem. I found one of these in a major js testing suite's tests...

It's why this tooling is worth applying...